src/System.Net.NameResolution/src/System/Net/NameResolutionPal.Windows.Uap.cs

```diff
@@ -0,0 +1,11 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Net
+{
+    internal static partial class NameResolutionPal
+    {
+        private static bool GetAddrInfoExSupportsOverlapped() => false;
```

Comment:
GetAddrInfoEx and GetAddrInfoExCancel are available in UAP no? LoadLibraryEx was used to test for unsupported windows version, but this may not be needed for UAP and it could return true.

https://docs.microsoft.com/en-us/uwp/win32-and-com/win32-apis

Reply:
Yeah I think you're right. Once I get this checked in I'll verify and flip to true.